apps/mochi-web/components/settings/general/DefaultMessage.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Button,
+  FormLabel,
+  IconButton,
+  Switch,
+  Typography,
+} from '@mochi-ui/core'
+import { EditLine, TrashBinLine } from '@mochi-ui/icons'
+import React from 'react'
+import {
+  Control,
+  Controller,
+  UseFormWatch,
+  useFieldArray,
+} from 'react-hook-form'
+import clsx from 'clsx'
+import { GeneralFormValue } from './types'
+import { MessageModal, actionList } from './MessageModal'
+
+interface Props {
+  control: Control<GeneralFormValue>
+  watch: UseFormWatch<GeneralFormValue>
+}
+
+export const DefaultMessage = ({ control, watch }: Props) => {
+  const { fields, append, remove, update } = useFieldArray({
+    control,
+    name: 'defaultMessage',
+  })
+  const enableDefaultMessage = watch('enableDefaultMessage')
+
+  return (
+    <div className="flex flex-col w-full max-w-md space-y-2">
+      <div className="flex items-center justify-between">
+        <FormLabel>Default message</FormLabel>
+        <Controller
+          name="enableDefaultMessage"
+          control={control}
+          render={({ field: { value, onChange, ...rest } }) => (
+            <Switch
```

Comment:
Hide the content instead of blurring it

<img width="598" alt="image" src="https://github.com/consolelabs/mochi-ui/assets/12707960/f0a0ef38-a4a4-4f79-a636-d6343bf6d82f">

Reply:
I will fix this in another pr